scripts/library-vmaf-eval.ps1

```
@@ -0,0 +1,66 @@
+    exit 1
+}
+
+$ScriptDir = Split-Path -Parent $MyInvocation.MyCommand.Path
+
+function Resolve-Path([string]$p) {
+    if ($p -notmatch '^[\\/]') {
+        Join-Path (Join-Path $ScriptDir '..') $p
+    } else {
+        $p
+    }
+}
```

PSScriptAnalyzer (1.25.0)

[warning] 'Resolve-Path' is a cmdlet that is included with PowerShell (version core-6.1.0-windows) whose definition should not be overridden

(PSAvoidOverwritingBuiltInCmdlets)